### PR TITLE
i18n(de): update guides/deploy/vercel.mdx

### PR DESCRIPTION
**Pull request by maxzirps**
#### Description (required)

updated german translation of src/content/docs/de/guides/deploy/vercel.mdx

#### Related issues & labels & PRs

#8217 
#8167 
#6267

### CODE REVIEW

**Comment by jsparkdev** [comment]
Feedback and questions about the code you've modified.

1. line 26 seems to be missing the inline code highlighting.
2. on line 86, the CLI asks a question in English, if you have a specific reason for translating the CLI's question, please let me know.

Thank you for your contribution.

File: src/content/docs/de/guides/deploy/vercel.mdx [hunk 1]
```diff
@@ -75,14 +85,11 @@ Nachdem dein Projekt importiert und veröffenlicht wurde, erzeugen alle nachfolg
...
 ### Projektkonfiguration mit vercel.json
 
 Du kannst `vercel.json` verwenden, um das Standardverhalten von Vercel außer Kraft zu setzen und zusätzliche Einstellungen zu konfigurieren. Du kannst zum Beispiel Header an die HTTP-Antworten deiner Veröffentlichungen anhängen.
 
-📚 Erfahre mehr über [Vercel's Projektkonfiguration](https://vercel.com/docs/project-configuration).
-
-### Auf Astro 2.0 aktualisieren
-
-Astro v2.0 stellt die Unterstützung für Node 14 ein. Stelle also sicher, dass dein Projekt **Node `16.12.0` oder höher** verwendet. Du kannst [die Node.js-Version](https://vercel.com/changelog/node-js-version-now-customizable-in-the-project-settings), die während des Build-Schritts und der Serverless-Funktionen verwendet wird, auf der Seite Allgemein unter Projekteinstellungen festlegen.
+<ReadMore>Erfahre mehr über [Vercel's Projektkonfiguration](https://vercel.com/docs/project-configuration).</ReadMore>
```
Comment (trueberryless) [inline src/content/docs/de/guides/deploy/vercel.mdx:95]:
```suggestion
<ReadMore>Erfahre mehr über [Vercel's Projektkonfiguration](https://vercel.com/docs/project-configuration).</ReadMore>

```

**Review by trueberryless — APPROVED** [review]
LGTM 🎉 

Just create a newline at the EOF (end of file) because if consistency please 🥺!

**Comment by maxzirps** [comment]
Nice findings @jsparkdev 

Thanks for the feedback. Updated the code

**Review by jsparkdev — APPROVED** [review]
LGTM!

**Pull request merged**